tests/unit/python/foglamp/services/core/api/test_common_ping.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test rest server api for python/foglamp/services/core/api/common.py """
+
+from aiohttp import web
+import json
+import pytest
+
+from foglamp.services.core import routes
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("api", "common")
+async def test_ping(test_server, test_client, loop):
+    app = web.Application()
+    # fill route table
+    routes.setup(app)
```

Comment:
Shouldn't we use only GET ping route here instead of all routes? I think better to load routes file-wise or common code

```
from foglamp.services.core.api import common as api_common
app.router.add_route('GET', '/foglamp/ping', api_common.ping())
```

Reply:
I would agree here. Do you see any problems maintaining routes in two places (source and test)?

Reply:
My opinion is, that will add an overhead of another test to that static def (routes setup) test And will be an extra baggage to import and maintain.